src/api/app/views/layouts/webui/responsive_ux/_navigation.html.haml

```diff
@@ -14,12 +14,18 @@
             %i.fas.fa-tasks.fa-lg
             - if tasks.positive?
               %span.badge.badge-primary.align-text-top= tasks
-        .toggler{ data: { toggle: 'offcanvas' } }
-          = image_tag_for(User.session, size: 40, custom_class: 'rounded-circle bg-light')
+        - if content_for?(:contextual_actions)
+          .toggler
+            = link_to('javascript:void(0)', class: 'nav-link text-light', alt: 'Contextual Actions', data: { toggle: 'contextual-actions' }) do
+              %i.fas.fa-th.fa-lg
```

Comment:
What about using `ellipsis-v` instead? I think nowadays most of the people is used to the three dots icon to find extra actions on the current element. It would look like:

![3dots](https://user-images.githubusercontent.com/2581944/76230944-c44dd000-6224-11ea-9e5d-2028e730fb99.png)